firmware/cmake/embedded.cmake

```diff
@@ -250,5 +250,7 @@ function(embedded_image
             WORKING_DIRECTORY ${REPO_ROOT_DIR}
             DEPENDS ${GENERATE_IMAGE_SCRIPT} ${APP_HEX_PATH} ${BOOT_HEX_PATH}
     )
+
+    log_bus_load(${IMAGE_HEX})
```

Comment:
why log it after the image is created vs putting this alongside the other jsoncan generation code?

Reply:
I didn't put it with jsoncan builds because CMake cache json CAN build and if json CAN build is cache, then it doesn't run the script which make the message does not always print. I know that putting line in here is a little weird and I just came up with a better way to do it.